WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -238,6 +241,12 @@ class PostListMainViewModel @Inject constructor(
     fun onTabChanged(position: Int) {
         val currentPage = POST_LIST_PAGES[position]
         updateViewStateTriggerPagerChange(isFabVisible = FAB_VISIBLE_POST_LIST_PAGES.contains(currentPage))
+
+        AnalyticsUtils.trackWithSiteDetails(
+                POST_LIST_TAB_CHANGED,
+                site,
+                mutableMapOf("selected_tab" to currentPage.toString() as Any)
```

Comment:
I think we can simplify this a bit like so:

```suggestion
                mapOf("selected_tab" to currentPage.toString())
```

I'd also suggest moving the "selected_tab" and the "author_filter_selection" to some kind of constant.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9654/commits/7c5243c4a8977df86824dd4ee704cf77e75ba2e7

Reply:
@malinajirka It looks like `as Any` conversions are still there 😢

Reply:
Sorry 😞 , fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9654/commits/22cc80c6847192ea3b4073e81ad303db7c919f47